annotations/helm-annotations/src/main/resources/NOTES.txt

```diff
@@ -0,0 +1,43 @@
+# Dekorate Autogenerated Notes
+
+To access the Helm annotations or properties you just need to have the following dependency in your
+class path:
+
+    <dependency>
+      <groupId>io.dekorate</groupId>
+      <artifactId>helm-annotations</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+Compile the project using:
+
+    mvn clean install
+
+You can find the generated Helm artifacts under: `target/classes/META-INF/dekorate/helm/<chart name>/` that should look like:
+- Chart.yaml
+- values.yaml
+- templates/*.yml the generated resources by Dekorate
+
+How can we use it?
+
+First, make sure you have logged into a cluster with Helm enabled (in OpenShift clusters, Helm is enabled by default).
```

Comment:
This sentence is not correct. Since helm v3, no server component is deployed anymore on k8s or openshift. So, currently it is only needed that helm v3 is installed locally.

Reply:
Updated to:
```
First, make sure you have installed [the Helm command line](https://helm.sh/docs/intro/install/) and have connected into a cluster.
```